chapters/simple_exec.md

```diff
@@ -1,4 +1,4 @@
-# Built-in Simple Executors and Reserved `uses` in Jina
+# Built-in Simple Executors and Reserved `uses`
```

Comment:
Also Title Case please  :)

Reply:
Again, it already is according to titlecase.com :)